### ISSUE TITLE
Performance of CPFT

### ISSUE THREAD

**Issue by FlashCoding** [ZIZUN/CPFT#1]
Hi ZIZUN, 

Thanks for re-implementing CPFT! I am currently running your code but it seems I cannot obtain the performance that the paper claims. For example, for HWU64 5-shot, I can only get ~77% acc on the test set (82.03% in the paper). I first pretrained for 500 epochs and the final pretraining loss is 5.315.
May I know if you can reach a similar performance with the paper and how you set up the pretraining epochs? Thanks!

**Comment by Green-li** [comment]
> Hi ZIZUN,
> 
> Thanks for re-implementing CPFT! I am currently running your code but it seems I cannot obtain the performance that the paper claims. For example, for HWU64 5-shot, I can only get ~77% acc on the test set (82.03% in the paper). I first pretrained for 500 epochs and the final pretraining loss is 5.315. May I know if you can reach a similar performance with the paper and how you set up the pretraining epochs? Thanks!

Hi @FlashCoding , have you reproduced the results of the CPFT?

**Comment by ZIZUN** [comment]
@FlashCoding @Green-li 
It seems too overfitted. When I experimented before, the performance was reproduced exactly as described in the paper. In paper says, 'We pre-train the combined intent datasets without test sets in the contrastive pre-training stage for 15 epochs.' Please make sure to check the hyperparameters properly. Sorry for the late reply. Thanks.
